l10n_ch_payment_slip/i18n/fr.po

```diff
@@ -23,22 +23,22 @@ msgstr "Données BVR"
 
 #. module: l10n_ch_payment_slip
 #: field:res.company,bvr_delta_horz:0
-msgid "BVR Horz. Delta (px)"
+msgid "BVR Horz. Delta (inch)"
 msgstr "Décalage horizontal global en pixels"
```

Comment:
The translation still says "en pixels", should be "en pouces" (same comment for the other translations in the file)